module/VuFind/src/VuFind/Search/Factory/BrowZineBackendFactory.php

```diff
@@ -109,7 +112,10 @@ protected function createBackend(Connector $connector)
     protected function createConnector()
     {
         // Validate configuration:
-        if (empty($this->browzineConfig->General->access_token)) {
+        if (
```

Comment:
Why not move the getSecretFromConfigOrSecretFile check higher in the function? Then you can check if that result is empty instead of redundantly checking two different configuration settings.